src/test/java/org/jabref/logic/database/DuplicateCheckTest.java

```diff
@@ -525,7 +525,7 @@ void compareOfTwoEntriesWithSameContentAndMixedLineEndingsReportsNoDifferences()
     }
 
     /**
-     * Book entries can have the same ISBN due to different chapters. The Test fails as crossref identifies both entries as the same.
+     * Journal articles can have the same ISBN due to the journal has one unique ISBN, but hundrets of different articles.
```

Comment:
There is a small typo here
```suggestion
     * Journal articles can have the same ISBN due to the journal has one unique ISBN, but hundreds of different articles.
```